### PR TITLE
Ledger design

### PR DESCRIPTION
**Pull request by georgeburduhos**

### CODE REVIEW

File: src/screens/connect-hardware-wallet/styles.ts [hunk 1]
```diff
@@ -33,14 +33,58 @@ export default (theme: ITheme) =>
...
+            color: theme.colors.accent
+        },
+        divider: {
+            flex: 1,
+            height: 2,
+            alignSelf: 'center',
+            backgroundColor: theme.colors.cardBackground
+        },
+        dividerSelected: {
+            backgroundColor: theme.colors.accent + 'AA'
```
Comment (krisboit) [inline src/screens/connect-hardware-wallet/styles.ts]: this is not ok, do we need a new color ?

File: src/core/wallet/hw-wallet/ledger/config.ts [hunk 2]
```diff
@@ -1,42 +1,54 @@
-import { Blockchain } from '../../../blockchain/types';
 import { HWConnection } from '../types';
 
 interface ILedgerTransportConfig {
```
[the remark this answers is not in the document]
Comment (georgeburduhos) [reply src/core/wallet/hw-wallet/ledger/config.ts]: Based on new steps before starting the connect the config changed. In the new version first you choose the blockchain then ledger model and based on this 2 options the list o available transports are shown

File: src/core/wallet/hw-wallet/ledger/transport-factory.ts [hunk 3]
```diff
@@ -3,19 +3,42 @@ import { BLE } from './transport/transport-ble';
 import { USB } from './transport/transport-usb';
 import { Platform } from 'react-native';
 import { ledgerConfig } from './config';
+import { Blockchain } from '../../../blockchain/types';
 
 export class TransportFactory {
     public static async get(
+        blockchain: Blockchain,
```
[the remark this answers is not in the document]
Comment (georgeburduhos) [reply src/core/wallet/hw-wallet/ledger/transport-factory.ts]: Correct. I had an extra verification for config but its not necessary anymore

File: src/screens/connect-hardware-wallet/styles.ts (hunk 1 above)
Comment (georgeburduhos) [reply src/screens/connect-hardware-wallet/styles.ts]: I will change this once the design fixes that Novi added will be in master. He reduced the number of gray.

File: src/core/wallet/hw-wallet/ledger/config.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (krisboit) [reply src/core/wallet/hw-wallet/ledger/config.ts]: ok, but basically you don't add new stuff, you just do a restructure, we are repeating a lot of information..

**Pull request merged**